Src/Fido2/AttestationFormat/Apple.cs

```diff
@@ -0,0 +1,148 @@
+﻿using System;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Security.Cryptography.X509Certificates;
+using Asn1;
+using Fido2NetLib.Objects;
+using PeterO.Cbor;
+
+namespace Fido2NetLib.AttestationFormat
+{
+
+    internal class Apple : AttestationFormat
+    {
+        private readonly IMetadataService _metadataService;
+        public Apple(CBORObject attStmt, byte[] authenticatorData, byte[] clientDataHash, IMetadataService metadataService)
+            : base(attStmt, authenticatorData, clientDataHash)
+        {
+            _metadataService = metadataService;
+        }
+
+        public override void Verify()
+        {
+            // Reference: https://developer.apple.com/documentation/devicecheck/validating_apps_that_connect_to_your_server
+
+            // Verify that attStmt is valid CBOR conforming to the syntax defined above and
+            // perform CBOR decoding on it to extract the contained fields.
+            if (0 == attStmt.Keys.Count || 0 == attStmt.Values.Count)
+                throw new Fido2VerificationException("Attestation format apple must have attestation statement");
+
+            if (null == Alg || true != Alg.IsNumber)

```

Comment:
Doesn't `Alg == null || Alg.IsNumber !=  true` read better?

Reply:
Yeah, maybe. This is code copied&modified from the "packed" attestation format, so I think I'm choosing to keep this code in the style the original author used.

Reply:
`Alg == null || !Alg.IsNumber` reads even better. :)

Reply:
Then again, I assume the style used here is consistent with the rest of the lib, which counts toward getting a future PR accepted.